trin-storage/src/versioned/id_indexed_v1/store.rs

```diff
@@ -811,12 +815,15 @@ mod tests {
         assert!(!store.has_content(&id)?);
         let usage_stats = store.usage_stats();
 
-        store.insert(&key, value.clone())?;
+        store.insert(&key, value.clone().into())?;
```

Comment:
Instead of calling `value.into()` here and in many places below, we should refactor `generate_key_value` and `generate_key_value_with_content_size` so they return `RawContentValue`